content/en/tracing/services/resource_page.md

```diff
@@ -61,21 +61,24 @@ Use the top right percentile selectors to zoom into a given percentile, or hover
 
 {{< img src="tracing/visualization/service/latency_distribution_sidebar.png" alt="latency distribution selector" style="width:50%;">}}
 
-## Dependency Map
+## Dependency Map with Navigator
 
-You can also view a map of all of a resource's upstream and downstream service dependencies. With the Dependency Map, you can quickly see the flow of services with spans that go through the specific resource (such as endpoints or database queries) end-to-end.
+You can also view a map of all of a resource's upstream and downstream service dependencies. With the Dependency Map Navigator, you can quickly see the flow of services with spans that go through the specific resource (such as endpoints or database queries) end-to-end, along with their request count.
 
 The dependency map is only available for resources containing service entry spans.
 
-{{<img alt="resource dependency map" src="tracing/visualization/resource/resource_dependency_map.png" style="width:100%;">}}
+{{< img src="tracing/visualization/resource/dependency-map-navigator.png" alt="dependency map navigator" style="width:100%;">}}
 
-Hover over a node to view metrics of each service including requests/second, error rate, and average latency.
+Hover over a node to view metrics of each service including requests/second, error rate, and average latency. Click on a node to open a context menu with options to view the Service Page, related traces, and more.
 
 The highlight color of the node indicates the service's [monitor status][5]. If a service has more than one configured monitor, the status of the most severe monitor is shown.
 
-{{<img src="tracing/visualization/resource/resource_dependency_map_hover.mp4" video="true" alt="hovering and clicking a resource dependency map node" style="width:100%;">}}
+{{< img src="tracing/visualization/resource/dependency-navigator.mp4" video="true" alt="hovering and clicking on the list" style="width:100%;">}}
+
+A service has load amplification if it's receiving more than 100% of the requests received by the selected resource upstream. Services with call paths highlighted in orange have load amplification, and the amplification (multiplier) is shown in the list on the panel. The amplification is calculated based on the requests received by the resource (shown on the map in the image below), and the requests received by the service (shown in the service node on the map). By clicking on a service in the list, you can see the spans contributing to the amplification.
```

Comment:
```suggestion
A service has load amplification if it's receiving more than 100% of the requests received by the selected resource upstream. Services with call paths highlighted in orange have load amplification, and the amplification multiplier is shown in the list on the panel. The amplification is calculated based on the requests received by the resource (shown highlighted on the map in the image below), and the requests received by its downstream service nodes (also shown on the map). By clicking on a service in the list, you can see the spans contributing to the amplification.
```

Reply:
Minor update to clarify the image, but let me know if any of this wording is incorrect

Reply:
"and the requests received by the downstream service (shown inside the downstream service node on the map)"

Reply:
Sounds good to me, updated the suggestion